docs/usage/Installations.md

```diff
@@ -122,6 +127,23 @@ status:
       namespace: default
 ```
 
+## Context
+
+The context is a configuration resource containing shared configuration for installations.
+This config can contain the repository context, registry pull secrets, etc. . For detailed documentation see [./Context.md](./Context.md).
+
+Installations *MUST* reference a context by its name in `.spec.context` (it is defaulted to the `default` context if not defined).
```

Comment:
If a `context` key is required, we loose backwards-compatibility or am I reading this wrong?
Will it be allowed to ommit the `spec.context`key, which then would default to the `default` context,
or do we expect at least a `context: ""`?

Reply:
you're right the wording is a bit confusing.
Id the context is omitted the key is defaulted to `"default"`